src/rest-server/src/models/v2/job/k8s.js

```diff
@@ -339,6 +339,25 @@ const generateTaskRole = (taskRole, labels, config) => {
                   name: 'GANG_ALLOCATION',
                   value: gangAllocation,
                 },
+                // Pass user token to runtime to give runtime permission to call rest server
+                // Actually we should provide service token for kube-runtime and do not let
+                // runtime personate as a real user.
+                {
+                  name: 'PAI_USER_TOKEN',
```

Comment:
put it in `runtime-env.js`

Reply:
runtime-env is used for app container, not init container. We need the token in init container